endpoints.json

```diff
@@ -25,5 +25,19 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves an array of specified article",
```

Comment:
we can just send back the singluar article object as we know there will only ever be one - we dont need it to be in an array